lib/ansiblelint/rules/MetaTagValidRule.py

```diff
@@ -57,3 +58,23 @@ def matchplay(self, file, data):
                                 "{}, invalid: '{}'".format(msg, tag)))
 
         return results
+
+
+META_TAG_VALID = '''
+galaxy_info:
+    galaxy_tags: ['database', 'my s q l', 'MYTAG']
+    categories: 'my_category_not_in_a_list'
+'''
+
+if "pytest" in sys.modules:
+
+    import pytest
```

Comment:
Now this looks scary...

Reply:
It works. I can also ask few pytest cores for feeback if you really want. It is dead code in production and it is loaded when in testing. If you think a little bit it is kinda similar with how we do type checking.